packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.mdx

```diff
@@ -0,0 +1,114 @@
+      onClick: () => {
+        window.open(
+          'https://github.com/coveo/ui-kit/blob/master/packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.ts',
+          '_blank'
+        );
+      },
+    },
+  ]}
+  of={AtomicCommerceProductListStories.Default}
+/>
+
+## Usage
+
+This component is typically placed within the `products` section of the layout.
+
+```html
+<atomic-commerce-interface>
+  ...
+  <atomic-commerce-layout>
+    ...
+    <atomic-layout-section section="main">
+      ...
+      <atomic-layout-section section="products">
+        <atomic-commerce-product-list></atomic-commerce-product-list>
+      </atomic-layout-section>
+    </atomic-layout-section>
+  </atomic-commerce-layout>
+</atomic-commerce-interface>
+```
+
+The components supports three `display` modes:
+
+- `grid` (default)
+- `list`
+- `table`
+
+When [displaying products as a list or grid](https://docs.coveo.com/en/atomic/latest/usage/displaying-results/list-or-grid/), the component applies a basic template by default. You can also [define custom templates](https://docs.coveo.com/en/atomic/latest/usage/displaying-results/#defining-a-result-template).
+
+```html
+<atomic-commerce-product-list>
+  <atomic-product-template>
+    <template>
+      <!-- ... -->
+    </template>
+  </atomic-product-template>
+</atomic-commerce-product-list>
+```
+
+When [displaying products as a table](https://docs.coveo.com/en/atomic/latest/usage/displaying-results/table/), there is no default template. You must define a template that includes at least one `atomic-table-element` component, which serve as columns.
+
+```html
+<atomic-commerce-product-list display="table">
+  <atomic-product-template>
+    <template>
+      <atomic-table-element label="Product">
+        <atomic-product-link></atomic-product-link>
+      </atomic-table-element>
+      <atomic-table-element label="ID">
+        <atomic-product-text field="permanentid"></atomic-product-text>
+      </atomic-table-element>
+      <atomic-table-element label="Price">
+        <atomic-product-price></atomic-product-price>
+      </atomic-table-element>
+    </template>
+  </atomic-product-template>
+</atomic-commerce-product-list>
+```
+
+## Examples
+
+### Grid display mode with custom template
+
+<Canvas of={AtomicCommerceProductListStories.GridDisplayWithTemplate} />
+
+### List display mode with custom template
+
+<Canvas of={AtomicCommerceProductListStories.ListDisplayWithTemplate} />
+
+### Table display mode
+
+<Canvas of={AtomicCommerceProductListStories.TableDisplay} />
+
+## Reference
+
+<ArgTypes />
+
+### Methods
+
+#### `setRenderFunction`
+
+Sets a rendering function to bypass the standard HTML template mechanism for rendering results. You can use this function while working with web frameworks that don't use plain HTML syntax, e.g., React, Angular, or Vue.
+
+Do not use this method if you integrate Atomic in a plain HTML deployment.
+
```

Comment:
An example (or a link to an example) might be nice.

Reply:
No sadly I don't see any on the docs site. Only reference docs, which don't include examples.
<img width="729" alt="Screenshot 2025-06-11 at 9 23 40 AM" src="https://github.com/user-attachments/assets/fcee5e67-d33c-46bd-af95-0a3c6f40aaf5" />